### PR TITLE
docs(Select): Update docs for Select new API: labelRender

### PR DESCRIPTION
**Pull request by xliez**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

rc-select 组件新增 API：https://github.com/react-component/select/pull/995

### 💡 Background and solution

rc-select 组件新增 API，同步更新 antd 文档

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Update docs for Select new API: labelRender      |
| 🇨🇳 Chinese |    更新 Select 组件新 API: labelRender 的文档     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
labelRender 在 antd 哪个版本新增的？我怎么在更新日志里没找到？

**Comment by xliez** [comment]
> labelRender 在 antd 哪个版本新增的？我怎么在更新日志里没找到？

在 rc-select  [14.12.0](https://github.com/react-component/select/commit/72f6eac730e0c47884eced37b39180a584a3e3ac)

**Comment by li-jia-nan** [comment]
PR title 用英文哈

**Comment by xliez** [comment]
> PR title 用英文哈

👌

**Comment by afc163** [comment]
需要在 Changelog 补充相关改动。

File: components/select/index.zh-CN.md [hunk 1]
```diff
@@ -118,6 +119,7 @@ return (
 | status | 设置校验状态 | 'error' \| 'warning' | - | 4.19.0 |
 | suffixIcon | 自定义的选择框后缀图标。以防止图标被用于其他交互，替换的图标默认不会响应展开、收缩事件，可以通过添加 `pointer-events: none` 样式透传。 | ReactNode | `<DownOutlined />` |  |
 | tagRender | 自定义 tag 内容 render，仅在 `mode` 为 `multiple` 或 `tags` 时生效 | (props) => ReactNode | - |  |
+| labelRender | 自定义当前选中的 label 内容 render | (label: ReactNode) => ReactNode | - |  |
```
Comment (afc163) [inline components/select/index.zh-CN.md:122]:
```suggestion
| labelRender | 自定义当前选中的 label 内容 render | (label: ReactNode) => ReactNode | - | 5.15.0 |
```

Comment (xliez) [reply components/select/index.zh-CN.md]: 已补充

**Comment by afc163** [comment]
图片

**Comment by afc163** [comment]
ping @xliez

**Review by afc163 — APPROVED** [review]

**Pull request merged**